app/views/admin/shared/_tabnav_product.html.haml

```diff
@@ -15,7 +15,11 @@
   - if @product.is_a?(Instrument)
     = tab "Restrictions", edit_facility_price_group_product_path(current_facility, @product), (secondary_tab == "restrictions")
     = tab "Reservations", facility_instrument_schedule_path(current_facility, @product), (secondary_tab == "reservations")
-  = tab "Docs", [current_facility, @product, :file_uploads, file_type: "info"], (secondary_tab == "documentation")
+
+  = tab "Docs",
+    [current_facility, @product, :file_uploads, file_type: "info"],
```

Comment:
forgive me if I'm missing something, did anything change here, or is it just formatting?